packages/fast-components-react-base/src/viewport-positioner/viewport-positioner.stories.tsx

```diff
@@ -1,53 +1,193 @@
 import { storiesOf } from "@storybook/react";
 import React from "react";
-import ViewportPositioner from "./";
-import { AxisPositioningMode } from "./viewport-positioner.props";
+import ViewportPositioner, { AxisPositioningMode, ViewportPositionerProps } from "./";
 import Foundation from "@microsoft/fast-components-foundation-react";
+import { ViewportPositionerVerticalPosition } from "./viewport-positioner.props";
+import { isNil } from "lodash-es";
 
 const anchorElement: React.RefObject<HTMLDivElement> = React.createRef<HTMLDivElement>();
 
-const rootElement: React.RefObject<HTMLDivElement> = React.createRef<HTMLDivElement>();
+interface TestViewportProps {
+    positionerProps: ViewportPositionerProps;
+}
 
-storiesOf("Viewport Positioner", module).add("Default", () => (
-    <div
-        ref={rootElement}
-        style={{
-            height: "400px",
-            width: "400px",
-            overflow: "scroll",
-        }}
-    >
-        <div
-            style={{
-                height: "600px",
-                width: "600px",
-                padding: "250px",
-                background: "blue",
+class TestViewport extends React.Component<TestViewportProps, {}> {
+    private rootElement: React.RefObject<HTMLDivElement> = React.createRef<
+        HTMLDivElement
+    >();
+
```

Comment:
You shouldn't need this constructor

Reply:
Actually I do.  There is an instanciation order issue where the ref gets passed into one component before the other has initialized it.